app/views/planning_applications/validation/validation_requests/index.html.erb

```diff
@@ -23,7 +25,7 @@
     <% if @active_validation_requests.any? %>
       <% if @planning_application.not_started? %>
         <p class="govuk-body">
-          The following requests will be sent when the application is invalidated.
+          <%= "The following requests will be sent when the application is invalidated." unless @planning_application.has_only_time_extension_requests? %>
         </p>
       <% end %>
       <% if post_validation_requests_index? %>
```

Comment:
It looks like this will lead to an empty `<p>` if the application is not started and it has only time extension requests. Probably not a big deal, but perhaps the if statement could be `if @planning_application.not_started? && !@planning_application.has_only_time_extension_requests?` instead?

Reply:
As if by magic, I realised that applying a scope as per your previous suggestion means I don't need that `unless` as the time extension requests are already excluded on line 25 :)